lib/generators/spree_elasticsearch/templates/index.yml.sample

```diff
@@ -0,0 +1,43 @@
+#If you wish to add your own custom analyzers to elasticsearch
+#modify this file.  The template shown below is what we use in our
+#application
+settings:
+  analysis:
+    analyzer:
+       search_analyzer:
+         type: "custom"
```

Comment:
I would get rid of quote marks everywhere, because YAML will load values to strings by default.